internal/controller/argocdcommitstatus_controller.go

```diff
@@ -450,40 +437,15 @@ func lookupArgoCDCommitStatusFromArgoCDApplication(mgr mcmanager.Manager) mchand
 
 			application := &argocd.Application{}
 
-			// if clusterName is empty, then cluster == local cluster
-			appKey := appRevisionKey{
-				clusterName: clusterName,
```

Comment:
It's kind of nice that the mc-runtime takes care of this cluster part of filtering.